node/pod.go

```diff
@@ -228,6 +210,9 @@ func (pc *PodController) updatePodStatus(ctx context.Context, podFromKubernetes
 		"old reason": podFromKubernetes.Status.Reason,
 	}).Debug("Updated pod status in kubernetes")
 
+	// If we update the pod status with a terminal pod status, then we need to go ahead and forcefully delete the pod
+	// from api server
```

Comment:
Are we supposed to do something here? I don't understand the placement of this comment.